nservicebus/rabbitmq/delayed-delivery.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Delayed Delivery
+summary: Describes the native delayed delivery implementation in the RabbitMQ transport
+component: Rabbit
+versions: '[4,]'
+---
+
+Starting with Version 4.3, the transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). The transport now creates an infrastructure inside the broker to natively handle delaying messages.
```

Comment:
Not only preferred, but will be consistent with other documents as well. I believe I mentioned it in both places, or at least that was my intention :)